lib/data-builder.js

```diff
@@ -67,8 +67,10 @@ function fillDebugData(data, err) {
     if ((p in data)) continue;
     data[p] = err[p];
   }
-  // NOTE err.stack is not an enumerable property
+  // NOTE `err.stack` and `err.message` is not an enumerable property
   data.stack = err.stack;
+  if (err.message)
```

Comment:
> the result will come back with `"message" : ""`

Hmm, I did was not aware of this edge case. In that case, I guess the following would be best?

```js
if (err.message && !data.message) 
  data.message = err.message;
```